content/docs/reference/common_attributes.md

```diff
@@ -0,0 +1,200 @@
+
+Example:
+
+[sourcecode:html]
+<amp-img
+    media="(min-width: 650px)"
+    src="wide.jpg"
+    width="466"
+    height="355" layout="responsive"></amp-img>
+<amp-img
+    media="(max-width: 649px)"
+    src="narrow.jpg"
+    width="527"
+    height="193" layout="responsive"></amp-img>
+[/sourcecode]
+
+For more information, see [Layout & Media queries](https://www.ampproject.org/docs/guides/responsive/control_layout).
+
+## noloading
+
+The `noloading` attribute indicates whether the "loading indicator" should be turned **off** for this element. Many AMP elements are white-listed to show a "loading indicator", which is a basic animation that shows that the element has not yet fully loaded. 
+
+Often used with: images, animations, videos, and ads
+
+Example: 
+
+[sourcecode:html]
+<amp-img src="card.jpg"
+    noloading
+    height="190"
+    width="297"
+    layout="responsive">
+</amp-img>
+[/sourcecode]
+
+## on
+
+The `on` attribute is used to install event handlers on elements. The events that are supported depend on the element.
+
+Often used with: lightboxes, sidebars, live lists, and forms
+
+Syntax:
+
+[sourcecode:text]
+eventName:targetId[.methodName[(arg1=value, arg2=value)]]
+[/sourcecode]
+
+Example:
+
+[sourcecode:html]
+<button on="tap:my-lightbox">Open lightbox</button>
+<amp-lightbox id="my-lightbox" layout="nodisplay">
+  ...
+</amp-lightbox>
+[/sourcecode]
+
+For more information, see  [Actions and Events in AMP](https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-events.md).
+
+## placeholder
+
+The `placeholder` attribute can be set on any HTML element, not just AMP elements. The placeholder attribute indicates that the element marked with this attribute acts as a placeholder for the parent AMP element. If specified, a placeholder element must be a direct child of the AMP element.
+
+By default, the placeholder is immediately shown for the AMP element, even if the AMP element's resources have not been downloaded or initialized. Once ready, the AMP element typically hides its placeholder and shows the content. The exact behavior with respect to the placeholder is up to the element's implementation.
+
+Often used with: images, animations, videos, and ads
+
+Example:
+
+[sourcecode:html]
+<amp-anim src="animated.gif" width="466" height="355" layout="responsive">
+  <amp-img placeholder src="preview.png" layout="fill"></amp-img>
+</amp-anim>
+[/sourcecode]
+
+For more information, see [Placeholders & fallbacks](https://www.ampproject.org/docs/guides/responsive/placeholders).
+
+## sizes
+
+All AMP elements that support the `responsive` layout, also support the `sizes` attribute. The value of the `sizes` attribute is a sizes expression as described in [img sizes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img), but extended to all elements, not just images.
```

Comment:
Same as above, would reword "img sizes" to something like the <img>-tag's sizes attribute.